README.md

```diff
@@ -4,8 +4,8 @@ teamsnap/emque-consuming](https://www.codeship.io/projects/83d08620-2023-0132-2b
 # Emque::Consuming
 
 Emque Consuming is a Ruby application framework that includes everything needed
-to create and run services capable of consuming messages from a message broker
-in a Pub/sub architecture.  Messages can be produced with the
+to create and run application capable of consuming messages from a message
```

Comment:
I’m experimenting with the structure in dozer. Let’s see how that turns out
and decide.


*Shane Emmons* | Director of Technology and Principal Architect
TeamSnap - The easiest way to manage your team
(810) 834-9542 | shane@teamsnap.com | What is TeamSnap?
<http://www.youtube.com/watch?v=fx0vRXYGijU>

On Tue, Feb 17, 2015 at 1:35 PM, Ryan Williams <notifications@github.com>
wrote:

> In README.md
> <https://github.com/teamsnap/emque-consuming/pull/28#discussion_r24840390>
> :
>
> > @@ -4,8 +4,8 @@ teamsnap/emque-consuming](https://www.codeship.io/projects/83d08620-2023-0132-2b
> >  # Emque::Consuming
> >
> >  Emque Consuming is a Ruby application framework that includes everything needed
> > -to create and run services capable of consuming messages from a message broker
> > -in a Pub/sub architecture.  Messages can be produced with the
> > +to create and run application capable of consuming messages from a message
>
> True, I'm all in favor of having it make sense on first read, and
> improving consistency.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/teamsnap/emque-consuming/pull/28/files#r24840390>.
>